R/cat_add_data.R

```diff
@@ -7,15 +7,113 @@
 #' sources.
 #'
 #' @param x an object of class [elic_cat].
-#' @param mechanism
+#' @param data_source either a [`data.frame`][base::data.frame] or
+#' [`tibble`][tibble::tibble], a string with the path to a _csv_ or _xlsx_ file,
+#' or anything accepted by the [read_sheet()][googlesheets4::read_sheet]
+#' function.
```

Comment:
Lines 10-13 should be removed since `data_source`is inherited from `elic_cont_add_data`.